mars_troughs/datapaths.py

```diff
@@ -17,19 +17,22 @@ class _DataPaths:
     DATA: Path = (Path(__file__) / ".." / "data").resolve()
     INSOLATION1: Path = DATA / "Insolation_5million_1.txt"
     INSOLATION2: Path = DATA / "TMP2" / "Insolation_5million_2.txt"
-    RETREAT: Path = DATA / "Retreat_data.txt"
-    RETREAT2: Path = DATA / "TMP2" / "Retreat_data_tmp2.txt"
-    TMP1: Path = DATA / "TMP_xz.txt"
+    #RETREAT: Path = DATA / "Retreat_data.txt"
+    #RETREAT2: Path = DATA / "TMP2" / "Retreat_data_tmp2.txt"
+    #TMP1: Path = DATA / "TMP_xz.txt"
```

Comment:
Do you want to delete these? We can always pull them out of git's history if we need to get them back.